Sources/Workspace/Workspace.swift

```diff
@@ -513,6 +529,10 @@ public class Workspace {
             RepositoryPackageConstraint(
                 container: dependency.repository, requirement: requirement))
 
+        // Add unversioned constraints for edited packages.
+        let currentManifests = loadDependencyManifests(rootManifests: rootManifests, engine: engine)
+        constraints += currentManifests.unversionedConstraints()
```

Comment:
was tihnking that myself. ive been reading through the tests and mock system there to decipher it a bit to set something up here. still working out a plan of attack (and spelunking the workspace code, which is pretty dense)